src/dev-core/src/app/app.component.html

```diff
@@ -5,16 +5,18 @@
 <section>
   <nav>
     <a routerLink="/">Home</a>
+    <a routerLink="/i18n">i18n</a>
+    <a routerLink="/typography">Typography</a>
+    <a routerLink="/lazy-load">Lazy Load</a>
+    <a routerLink="/layout">Layout &amp; Grid</a>
+    <a routerLink="/dropdown">Test Dropdown</a>
```

Comment:
Eeek. I had these in alphabetical order...

Could you put them back?

Reply:
Yes. Noticed this and fixed it.